src/components/dev-hub/card.js

```diff
@@ -129,6 +132,7 @@ const Card = ({
                 {!collapseImage && (
                     <ImageWrapper>
                         {image && <Image src={image} alt="" />}
+                        <CardBadge contentType={badge} />
```

Comment:
Can we keep this optional?
```suggestion
                        {badge && <CardBadge contentType={badge} />}
```

I noticed the twitch card on the index page has a little badge in the bottom left it shouldn't have

Reply:
Thanks for catching that!